build.gradle

```diff
@@ -61,16 +69,46 @@ repositories {
     mavenCentral()
 }
 
+// Write the plugin's classpath to a file to share with the tests
+task createClasspathManifest {
+  def outputDir = file("$buildDir/$name")
+
+  inputs.files sourceSets.main.runtimeClasspath
+  outputs.dir outputDir
+
+  doLast {
+    outputDir.mkdirs()
+    file("$outputDir/plugin-classpath.txt").text = sourceSets.main.runtimeClasspath.join("\n")
+  }
+}
+
+tasks.test.dependsOn tasks.createClasspathManifest
+if (findProject(':vendor') != null) {
```

Comment:
I am confused. It works fine for me. What's the issue you encountered and how can I reproduce it?